contracts/FlowCallbackScheduler.cdc

```diff
@@ -541,11 +548,6 @@ access(all) contract FlowCallbackScheduler {
                 return Status.Canceled
             }
 
-            // if no callbacks were yet canceled the status is executed
-            if self.canceledCallbacks.length == 0 {
-                return Status.Executed
-            }
```

Comment:
We have a bug here... the problem is that if we query for ID that is bigger than actually executed IDs it will return executed status. We must add another field that keeps track of latest executed ID, then we need to add a check `if id > latestExecutedID return Unknown`

Reply:
Having latest executed ID we also don't have to check for first canceled ID, we can just check if ID is in canceled ID list, and if it's lower than last executed ID. We should remove init the canceled with 0

Reply:
I don't think that is necessary. We can just check if `id > self.nextID` and return `nil` if it is